From: Harish Kasiviswanathan <Harish.Kasiviswanathan@amd.com>
Subject: [PATCH] kernel: Export mm_access
Date: Tue, 23 May 2017 16:17:29 -0400
Message-ID: <5a9607b4-d2ac-fc7b-e43b-1e73c2651071@amd.com>
Patch-mainline: Submitted, https://patchwork.kernel.org/patch/9744281/
References: bsc#1040550

Exporting mm_access, which is required for implementing accelerated
equivalents of process_vm_readv/process_vm_writev for GPU memory in KFD
(AMD's GPU compute driver). This allows us to apply all the same remote
process memory access policies.

Reviewed-by: Felix Kuehling <Felix.Kuehling@amd.com>
Acked-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Harish Kasiviswanathan <Harish.Kasiviswanathan@amd.com>
Acked-by: Takashi Iwai <tiwai@suse.de>

---
 kernel/fork.c |    1 +
 1 file changed, 1 insertion(+)

--- a/kernel/fork.c
+++ b/kernel/fork.c
@@ -835,6 +835,7 @@ struct mm_struct *mm_access(struct task_
 
 	return mm;
 }
+EXPORT_SYMBOL_GPL(mm_access);
 
 static void complete_vfork_done(struct task_struct *tsk)
 {
